docker-compose.env

```diff
@@ -57,6 +57,7 @@ NUM_CONFIRMATIONS=0
 FINALITY_CONFIRMATIONS=0
 RUN_TX_BATCH_SUBMITTER=true
 RUN_STATE_BATCH_SUBMITTER=true
+MAX_BATCH_SUBMISSION_TIME=240
```

Comment:
Ideally we get these documented in https://github.com/ethereum-optimism/batch-submitter